src/styles/base/_override.links.scss

```diff
@@ -1,49 +1,17 @@
-%link-inverse {
-  color: $color-white;
-
-  &:visited {
-    color: $color-white;
-  }
-
-  &:hover,
-  &:active {
-    color: hsla(0, 0%, 100%, 0.77);
-  }
-}
-
-%link,
-%link-darker {
-  color: $color-primary-alt-light;
-
-  &:visited {
-    color: $color-visited;
-  }
-
-  // Override visited with hover
-  &:hover,
-  &:active {
-    color: $color-primary-alt;
-    outline: 0 none;
-  }
-
-  &:focus {
-    box-shadow: 0 0 0 4px $focus-choice-color-highlight,
-      0 0 0 2px $focus-choice-color;
-    border-color: $focus-choice-color;
-    outline: none;
-  }
-
-  &.mct-link--inverse {
-    @extend %link-inverse;
-  }
-}
-
-a,
-.ds-c-link {
-  @extend %link;
-}
-
-.ds-base--inverse a:not(.ds-c-button),
-.ds-c-link--inverse {
-  @extend %link-inverse;
-}
+// Links
+// Default link
+$link-color: $color-primary;
+$link-visited-color: #701e3c;
```

Comment:
[question] should this hex be a color variable? Or why is this one not a variable but all others are?

Reply:
Yes absolutely, Thanks for the catch - I'll need to add this new color to the color override